app.py

```diff
@@ -28,7 +28,7 @@ def init_app(self):
         if self.engine.name == "tk-mari":
             # Mari doesn't have the concept of a current scene so this app shouldn't
             # provide any commands!
-            application_has_scenes = False
```

Comment:
Setting the flag to false meant that the rest of the method wasn't executed, but it wasn't clear from just setting the flag. Returning clarifies the intent.